src/Autocomplete/src/OptionsAwareEntityAutocompleterInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Autocomplete;
+
+/**
+ * Interface for classes that will have an "autocomplete" endpoint exposed with a possibility to pass additional options.
+ */
+interface OptionsAwareEntityAutocompleterInterface extends EntityAutocompleterInterface
```

Comment:
I'd rather have separate Interfaces, what do you think ?

Reply:
I do like creating "subtype" interfaces like this, but it's not a thing I'd fight for :D. I'm open for changing it.

Reply:
No strong opinion on this.

i'm often more concerned about "what could happen later" than i probably should... i'm working on it but habits are hard to change :) 

That's why i ask / raise questions but would never "require" changes :))